packaging/olm-manifests/manifests/strimzi-cluster-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1310 @@
+                      - name: STRIMZI_OPERATOR_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.namespace
+                    livenessProbe:
+                      httpGet:
+                        path: /healthy
+                        port: http
+                      initialDelaySeconds: 10
+                      periodSeconds: 30
+                    readinessProbe:
+                      httpGet:
+                        path: /ready
+                        port: http
+                      initialDelaySeconds: 10
+                      periodSeconds: 30
+                    resources:
+                      limits:
+                        cpu: 1000m
+                        memory: 384Mi
+                      requests:
+                        cpu: 200m
+                        memory: 384Mi
+            strategy:
+              type: Recreate
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: true
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  relatedImages:
+    - name: strimzi-operator
+      image: quay.io/strimzi/operator@sha256:3ad4a43c0915030a6ef2509b90fb77265a8fd340d247421c741f3cab71938ae9
+    - name: strimzi-kafka-bridge
+      image: quay.io/strimzi/kafka-bridge@sha256:bed07186355a4c04ee50e90048e3bd3a023264093a9ef28f0278cce44640e93f
+    - name: strimzi-jmxtrans
+      image: quay.io/strimzi/jmxtrans@sha256:dc0a597b2f47dce7c702c855c03adeb57f308f52221afabcddaba50c1c45618e
+    - name: strimzi-kaniko-executor
+      image: quay.io/strimzi/kaniko-executor@sha256:c812530c2ea981d3316c7544b180289abfbd9adf1dde6f1345692b8fb0a65cb0
+    - name: strimzi-kafka-260
+      image: quay.io/strimzi/kafka@sha256:bdd8b92a44a9f9317c9ff65d9d1c06bd10f3f43cb9777db24b08baf26ccbb836
+    - name: strimzi-kafka-261
+      image: quay.io/strimzi/kafka@sha256:ea3e01dc5cbde1e10e06a26a08263b3ff01a79156c1f174d897121f568a8b3c8
+    - name: strimzi-kafka-270
+      image: quay.io/strimzi/kafka@sha256:b429bc96e7f075f8fb679165b3aab67a7ad87b66b78595a3a325bf60f4a5bbfb
+  keywords:
+    - kafka
+    - messaging
+    - kafka-connect
+    - kafka-streams
+    - data-streaming
+    - data-streams
+    - streaming
+    - streams
+  labels:
+    name: strimzi-cluster-operator
+  links:
+    - name: Website
+      url: https://strimzi.io/
+    - name: Documentation
+      url: https://strimzi.io/documentation/
+    - name: Mailing list
+      url: https://lists.cncf.io/g/cncf-strimzi-users/topics
+    - name: Slack
+      url: https://cloud-native.slack.com/messages/strimzi
+    - name: GitHub
+      url: https://github.com/strimzi/strimzi-kafka-operator
+  maintainers:
+    - email: cncf-strimzi-users@lists.cncf.io
+      name: Strimzi
+  maturity: stable
+  provider:
+    name: Strimzi
+  replaces: 0.22.1
```

Comment:
How do you know what was the previous version?

Reply:
I just added a curl script to the generator which will set this to the latest released version from OperatorHub